site/name-changes.md

```diff
@@ -0,0 +1,170 @@
+
+This changes how operators and domains are created, and also changes the generated artifacts for operators and domains.
+
+We're not providing an upgrade tool or backwards compatibility with the previous names.  Instead, customers need to use the new script, inputs file and parameter names to recreate their operators and domains.
+
+This document lists the customer visible naming changes.  Also, the WebLogic Server Kubernetes Operator documentation has been updated.
+
+## Customer Visible Files
+
+### Files for Creating and Deleting Operators and Domains
+
+The following files are used to create the operator and to create and delete domains.
+
+| Previous File Name | New File Name |
+| --- | --- |
+| kubernetes/create-domain-job.sh | kubernetes/create-weblogic-domain.sh |
+| kubernetes/create-domain-job-inputs.yaml | kubernetes/create-weblogic-domain-inputs.yaml |
+| kubernetes/create-operator-inputs.yaml | kubernetes/create-weblogic-operator-inputs.yaml |
+| kubernetes/create-weblogic-operator.sh | same |
+| kubernetes/delete-domain.sh | kubernetes/delete-weblogic-operator-resources.sh |
+
+### Generated YAML Files for Operators and Domains
+
+The create scripts generate a number of YAML files that are used to configure the corresponding Kubernetes artifacts for the operator and the domains.
+Normally, customers do not use these YAML files.  However, customers can look at them.  They can also change the operator and domain configuration by editing these files and reapplying them.
+
+#### Directory for the Generated YAML Files
+
+Previously, these files were placed in the kubernetes directory (e.g. kubernetes/weblogic-operator.yaml).  Now, they are placed in per-operator and per-domain directories (since a Kubernetes cluster can have more than one operator and an operator can manage more than one domain).
+
+The customer must create a directory that will parent the per-operator and per-domain directories, and use the -o option to pass the name of that directory to the create script, for example:
+  mkdir /scratch/my-user-projects
+  create-weblogic-operator.sh -o /scratch/my-user-projects
+The pathname can either be a full path name, or a relative path name.  If it's a relative pathname, then it's relative to the directory of the shell invoking the create script.
+
+The per-operator directory name is:
+  <user project dir from -o>/weblogic-operators/<operator namespace from the input YAML file's namespace property>
+
+Similarly, the per-domain directory name is:
+  <user project dir from -o>/weblogic-domains/<domain uid from the input YAML file's domainUid property>
+
+#### What If I Mess Up Creating a Domain or Operator And Want To Do It Again?
+
+* Remove the resources that were created for the operator:
+  * kubectl delete -f weblogic-operator.yaml
+  * kubectl delete -f weblogic-operator-security.yaml
+* Remove the resources that were created for the domain:
+  * kubernetes/delete-weblogic-domain-resources.sh <domainUID>
```

Comment:
The operator delete should be moved down so that it's the last cleanup step after the domain deletes.    (The domain delete script starts with an attempt to trigger graceful WL server shutdowns by setting the domain yamls startup-control value to NONE, and this depends on the operator to initiate the shutdowns.   After this attempt, it goes into a mode where it blows alway all resources -- including any WL server pods that failed to shutdown after setting NONE.)

Reply:
good catch - I'll fix it on another branch since Ryan already merged this work.